.git-hooks/prepare-commit-msg

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#
+# Inspects branch name and checks if it contains a Jira ticket number (i.e. MOB-1234).
+# If yes, [MOB-123] will be added in the end of a commit message.
+#
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD 2>/dev/null)
```

Comment:
Does this also check for things like `feature/MOB-123`? Just asking out of curiosity.

Reply:
Exactly, this line just gets full branch name.
And full script can find task number even if branch name contains prefix and/or suffix